### PR TITLE
Flag to compile oneTBB without threads

### PR DESCRIPTION
**Pull request by JhaShweta1**
### Description 
_Add a comprehensive description of proposed changes_

Fixes # - _issue number(s) if exists_

- [ ] - git commit message contains an appropriate signed-off-by string _(see [CONTRIBUTING.md](https://github.com/oneapi-src/oneTBB/blob/master/CONTRIBUTING.md#pull-requests) for details)_

### Type of change

_Choose one or multiple, leave empty if none of the other choices apply_

_Add a respective label(s) to PR if you have permissions_

- [ ] bug fix - _change that fixes an issue_
- [X ] new feature - _change that adds functionality_
- [ ] tests - _change in tests_
- [ ] infrastructure - _change in infrastructure and CI_
- [ ] documentation - _documentation update_

### Tests

- [ ] added - _required for new features and some bug fixes_
- [X ] not needed

### Documentation

- [ ] updated in # - _add PR number_
- [ ] needs to be updated
- [ ] not needed

### Breaks backward compatibility
- [ ] Yes
- [ ] No
- [ ] Unknown

### Notify the following users
_List users with `@` to send notifications_

### Other information
-DEMSCRIPTEN_WITHOUT_PTHREAD = TRUE/FALSE

### CODE REVIEW

File: WASM_Support.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ emcmake cmake .. -DCMAKE_CXX_COMPILER=em++ -DCMAKE_C_COMPILER=emcc -DTBB_STRICT=
 cmake --build . <options>
 cmake --install . <options>
 ```
-
+To compile oneTBB without pthreads, please pass the flag "-DEMSCRIPTEN_WITHOUT_PTHREAD=true" in the emcmake command,. By default, oneTBB uses the pthreads.
 Where:
 
 * ``emcmake`` - a tool that sets up the environment for Emscripten*.
```
Comment (dnmokhov) [inline WASM_Support.md:43]: - `,.` → `.`
- Maybe put the new paragraph after the "emcmake, DCMAKE..."  bullet list?

**Review by sarathnandu — DISMISSED** [review]
LGTM!

File: WASM_Support.md [hunk 2]
```diff
@@ -34,10 +34,12 @@ To build the system, run:
 ```
 mkdir build && cd build
 emcmake cmake .. -DCMAKE_CXX_COMPILER=em++ -DCMAKE_C_COMPILER=emcc -DTBB_STRICT=OFF -DCMAKE_CXX_FLAGS=-Wno-unused-command-line-argument -DTBB_DISABLE_HWLOC_AUTOMATIC_SEARCH=ON -DBUILD_SHARED_LIBS=ON -DTBB_EXAMPLES=ON -DTBB_TEST=ON
+```
+To compile oneTBB without pthreads, please pass the flag "-DEMSCRIPTEN_WITHOUT_PTHREAD=true" in the above command. By default, oneTBB uses the pthreads.
```
Comment (aepanchi) [inline WASM_Support.md:38]:
```suggestion
To compile oneTBB without ``pthreads``, set the flag ``-DEMSCRIPTEN_WITHOUT_PTHREAD=true`` in the command above. By default, oneTBB uses the ``pthreads``.
```

**Review by dnmokhov — APPROVED** [review]
LGTM

**Pull request merged**